resources/access.rb

```diff
@@ -16,14 +16,14 @@
 # limitations under the License.
 #
 
-property :source,        String, required: true, default: 'pg_hba.conf.erb'
-property :cookbook,      String, default: 'postgresql'
-property :comment,       [String, nil], default: nil
 property :access_type,   String, required: true, default: 'local'
 property :access_db,     String, required: true, default: 'all'
 property :access_user,   String, required: true, default: 'postgres'
-property :access_addr,   [String, nil], default: nil
 property :access_method, String, required: true, default: 'ident'
+property :cookbook,      String,                 default: 'postgresql'
```

Comment:
I read the source code... And i don't know where we use this parameter. Is it really needed to have and being able to modify it ?

Reply:
Ah right, everywhere we use a template, we should set  this. 

if it isn't used I should fix that! It allows the user to bring their own template.

Reply:
Misread the code, my bad, sorry